schemas/panels/time-series/time-series.cue

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 #legend: {
-	position?: "bottom" | "right"
+	position: "bottom" | "right"
```

Comment:
@AntoineThebaud is this still all I need to do to make this change, will the new migration code automatically work?